CriteoPublisherSdk/Sources/Context/CRContextData.m

```diff
@@ -0,0 +1,49 @@
+//
+//  CRContextData.m
+//  CriteoPublisherSdk
+//
+//  Copyright © 2018-2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "CRContextData.h"
+
+@interface CRContextData ()
+
+@property(strong, nonatomic, readonly) NSDictionary<NSString *, id> *data;
+
+@end
+
+@implementation CRContextData
+
++ (NSString *)contentUrl {
```

Comment:
I am sorry, noticing this a bit late, but I thought at first it was a typed property on Context, i.e. `self.contextData.contentUrl = @"http://whatev.er"`.
The naming is confusing. Usually such constants are declared as top level one, like [this](https://developer.apple.com/documentation/storekit/skstoreproductparameteradnetworkidentifier?language=objc). In this case this should be something like: `NSString *const CRContextDataContentUrl`

Reply:
Done